### PR TITLE
added npy field to db... still working on other parts

### PR DESCRIPTION
**Pull request by gould-ann**
it looks like i made a lot of changes but i really didnt 🤷‍♀

### CODE REVIEW

**Comment by gould-ann** [comment]
This is ready to merge 👀

**Comment by katebrune** [comment]
i'm about to pr the api routes that i changed with the people db, should we merge this now and fix later or wait for that to get merged change the call in this and then merge?

File: FacialRecog.py
Comment (katebrune) [inline FacialRecog.py]: can probably get rid of this if we aren't using it

File: FacialRecog.py
Comment (katebrune) [inline FacialRecog.py]: when is this used?

File: FacialRecog.py
Comment (katebrune) [inline FacialRecog.py:119]: for this to be able to be plugged into RecogScript w/ minimal refactoring we need to to return known_names, known_encodings

File: app.py
Comment (katebrune) [inline app.py]: i like how u set this up

**Review by katebrune — CHANGES_REQUESTED** [review]

File: FacialRecog.py
Comment (gould-ann) [reply FacialRecog.py:119]: wait why would we return the encodings? detected_faces is just a true/false list

Comment (katebrune) [reply FacialRecog.py:119]: because it's gonna be plugged in where `scan_for_known_people` is currently in RecogScript

image

we already have code to do the comparing, we just need the faces and names to compare against

File: FacialRecog.py
Comment (rgoluch) [reply FacialRecog.py]: when did this get added/what is the point? cough..cough...comments...cough...cough

File: app.py
Comment (rgoluch) [inline app.py]: why is this deprecated atm?

**Review by rgoluch — COMMENTED** [review]
I would make sure that the core functionality of what we had isn't changing. If we change how we get to that functionality that's fine, but not what things are returning or expecting.

File: app.py
Comment (katebrune) [reply app.py]: we don't have/use user db @rgoluch

File: FacialRecog.py
Comment (katebrune) [inline FacialRecog.py:153]: `get_all_known()` -> `get_known_people()` after merging

File: FacialRecog.py
Comment (katebrune) [inline FacialRecog.py:149]: looks good

**Review by katebrune — APPROVED** [review]

File: tests/api_tests.py
Comment (katebrune) [inline tests/api_tests.py:59]: `/known` route doesn't exist anymore so this is gonna fail

**Pull request merged**
